package.json

```diff
@@ -9,14 +9,14 @@
   ],
   "scripts": {
     "build": "rollup -c",
-    "test": "npm-run-all test-headless eslint",
+    "prepare": "npm run build",
+    "test": "npm run build && npm-run-all test-headless eslint",
     "test-headless": "SELENIUM_BROWSER=chrome E2E_HEADLESS=1 ./scripts/wdio-suppress-exitcode",
     "test-chrome": "SELENIUM_BROWSER=chrome ./scripts/wdio-suppress-exitcode",
     "test-firefox": "SELENIUM_BROWSER=firefox ./scripts/wdio-suppress-exitcode",
     "test-inspect-brk": "E2E_WDIO_EXEC_ARGV=--inspect-brk npm run test-chrome",
     "test-verbose": "SELENIUM_BROWSER=chrome E2E_LOG_LEVEL=verbose wdio wdio.conf.js",
-    "eslint": "eslint src/ tests/",
-    "updtr": "updtr"
```

Comment:
`npx updtr` can be used instead

Reply:
Awesome stuff @mo with that release, thanks. For this minor thing about npx updtr ^ I meant we could drop this line, npx is like yarn, it runs things in node_modules/.bin directly, without the need to put them in npm scripts. Sorry I wasn't clear. Thanks, I didn't know that package, I use usually npm update